deployment/envgen/main.cpp

```diff
@@ -83,6 +83,10 @@ void usage()
   puts("          for thor slaves. A node for thor master is automatically added. ");
   puts("          If not specified or specified as 0, no thor nodes");
   puts("          are generated");
+  puts("   -espnodes <number of esp nodes>: Number of nodes to be generated  ");
+  puts("          If not specified 1 esp node is generated");
+  puts("          If 0 is specified then all available nodes specified with assign_ips flag ");
+  puts("          will be used, others 0 esp nodes will be generated");
```

Comment:
@garonsky Looks good, we might want to add that espnodes only works for more than one instance if you explicitly declare assign_ips esp <range>